charts_common/lib/src/chart/common/behavior/selection/select_nearest.dart

```diff
@@ -101,14 +101,13 @@ class SelectNearest<D> implements ChartBehavior<D> {
 
   bool _delaySelect = false;
 
-  SelectNearest(
-      {this.selectionModelType = SelectionModelType.info,
-      this.expandToDomain = true,
-      this.selectAcrossAllSeriesRendererComponents = true,
-      this.selectClosestSeries = true,
-      this.eventTrigger = SelectionTrigger.hover,
-      this.maximumDomainDistancePx,
-      this.hoverEventDelay}) {
+  SelectNearest({this.selectionModelType = SelectionModelType.info,
+    this.expandToDomain = true,
+    this.selectAcrossAllSeriesRendererComponents = true,
+    this.selectClosestSeries = true,
+    this.eventTrigger = SelectionTrigger.hover,
+    this.maximumDomainDistancePx,
+    this.hoverEventDelay}) {
```

Comment:
super unimportant but i think the formatting got weird because of the missing trailing comma

Reply:
I'll fix that -- there are a bunch of formatting issues in the package.